buildscripts/github/release-environment.yml

```diff
@@ -0,0 +1,6 @@
+name: llvmlite-release-env
+channels:
+  - numba
+  - defaults
+dependencies:
+  - llvmdev=11.*
```

Comment:
Should it be this?

```suggestion
  - llvmdev=14.*
```

Reply:
Yes, it should. I was working this before LLVM14 got merged.